gateway/modules/eventing/routine.go

```diff
@@ -1,19 +1,13 @@
 package eventing
 
-import (
-	"time"
-)
-
 func (m *Module) routineProcessIntents() {
-	ticker := time.NewTicker(10 * time.Second)
-	for t := range ticker.C {
+	for t := range m.ticker.C {
 		m.processIntents(&t)
 	}
 }
 
 func (m *Module) routineProcessStaged() {
-	ticker := time.NewTicker(10 * time.Second)
-	for t := range ticker.C {
+	for t := range m.ticker.C {
```

Comment:
@YourTechBud can we use a single ticker here? I think we require 2

Reply:
You are correct. We need to different tickers